src/entity/Entity.php

```diff
@@ -1474,7 +1474,7 @@ protected function sendSpawnPacket(Player $player) : void{
 			array_map(function(Attribute $attr) : NetworkAttribute{
 				return new NetworkAttribute($attr->getId(), $attr->getMinValue(), $attr->getMaxValue(), $attr->getValue(), $attr->getDefaultValue(), []);
 			}, $this->attributeMap->getAll()),
-			$this->getAllNetworkData(),
+			EntityMetadataProperties::convertProps($player->getNetworkSession()->getProtocolId(), $this->getAllNetworkData()),
```

Comment:
I personally think that the AddActorPacket should have this itself

Reply:
I was following the same standards as the AvailableCommandsPacket. Should I not?

Reply:
Well seems like i made the mistake there then 😅

Reply:
Resolved this for entity metadata, adding the same change for command parameters is out of this pr's scope